index.js

```diff
@@ -89,22 +60,62 @@ function processArgv(argv) {
       module = require(dirName);
       pkgJson = require(pkg);
     } catch (err) {
-      log(verbose, `Error loading module: ${pkg}`);
+      logger.verbose(`Error loading module: ${pkg}`);
     }
 
     if (module && typeof module.runCommand === 'function') {
       const pkgName = pkgJson.name || dirName;
 
       if (modulesCalled[pkgName]) {
-        log(verbose, `Multiple instances found. Skipping passing command to ${pkgName}`);
+        logger.verbose(`Multiple instances found. Skipping passing command to ${pkgName}`);
       } else {
-        log(verbose, `Passing command to ${pkgName}`);
+        logger.verbose(`Passing command to ${pkgName}`);
         module.runCommand(command, argv);
         modulesCalled[pkgName] = true;
       }
     }
   });
 
+  if (Object.keys(modulesCalled).length === 0) {
+    fatal(`No module found for ${command}`);
+  }
+}
+
+/**
+ * Processes an argv object.
+ * Reads package.json if it exists.
+ * @name processArgv
+ * @param [Object] argv
+ */
+function processArgv(argv) {
+  let command = argv._[0];
+  // Allow shorthand "-v" for version
+  if (argv.v) {
+    command = 'version';
+  }
+
+  // Allow shorthand "-h" for help
+  if (argv.h) {
+    command = 'help';
+  }
+
+  switch (command) {
+    case 'version':
+      require('./lib/version').logVersion(argv);
+      break;
+    case 'new':
+      require('./lib/new')(argv);
+      break;
+    case 'help':
+    case undefined:
+      require('./lib/help')(argv);
+      break;
+    default:
+      logger.verbose(`SKY UX processing command ${command}`);
```

Comment:
This was previously an `info`.  Did you want to intentionally change it to `verbose` or did it just get caught up in changing the others?

Reply:
:doh: